aws/nextflow-launch-templates.tf

```diff
@@ -29,3 +29,20 @@ resource "aws_launch_template" "nf_lt_bigdisk" {
   }
   update_default_version = true
 }
+
+resource "aws_launch_template" "nf_lt_auto_scaled_ebs" {
+  name = "nextflow-launchtemplate-auto-scaled-ebs"
+  tags = var.default_tags
+  # Amazon Linux 2 Kernel 5.10 AMI 2.0.20220426.0 x86_64 HVM gp2
+  image_id = "ami-0022f774911c1d690"
```

Comment:
Based on the instructions [here](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-optimized_AMI.html#ecs-optimized-ami-linux-variants.html) I ran `aws ssm get-parameters --names /aws/service/ecs/optimized-ami/amazon-linux-2/recommended` and got a different ami: `ami-061c10a2cb32f3491`. I don't know if this is maybe a region difference?